f8a_tagger/utils.py

```diff
@@ -24,10 +46,24 @@ def iter_files(path, ignore_errors=True):
         item = stack.pop()
 
         if os.path.isfile(item):
-            yield item
+            yield item, item
         elif os.path.isdir(item):
             for entry in os.listdir(item):
                 stack.append(os.path.join(item, entry))
+        elif item.startswith(('http://', 'https://')):
+            try:
+                content, suffix = _get_remote_resource(item)
+                temp_file = tempfile.NamedTemporaryFile(mode='w+t', delete=False, suffix=suffix)
```

Comment:
As far as I know, the default mode is `t`. Do we still need to specify it explicitly. ?

Reply:
Default is `w+b`, see docs https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile Anyway, stating it explicitly shouldn't harm anything.

Reply:
@fridex 👍